packages/fluentui/react-northstar/src/components/Table/Table.tsx

```diff
@@ -114,40 +124,57 @@ class Table extends UIComponent<WithAsProp<TableProps>> {
     return TableRow.create(header, {
       defaultProps: () => ({
         ...headerRowProps,
-        accessibility: accessibility.childBehaviors ? accessibility.childBehaviors.row : undefined,
+        accessibility: (accessibility && accessibility({ header: !!header })?.childBehaviors.row) || undefined,
```

Comment:
Don't forget to add `{...getA11yProps('row', {`

Reply:
Fixed